### PR TITLE
Feat/add icons

### PR DESCRIPTION
**Pull request by borislavkb**
-React-icons imported and added

### CODE REVIEW

File: src/components/Main.js [hunk 1]
```diff
@@ -1,19 +1,22 @@
 import "./Main.css";
 import ItemCard from "./ItemCard";
+import image01 from "../images/sunday.png";
+import image02 from "../images/red.png";
+import image03 from "../images/chocolate.png";
```
Comment (me-jterhorst) [inline src/components/Main.js:5]: You might want to do yourself a favour and name those images to something decriptive

File: src/pages/AddNewCake.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+  background: var(--color-background);
+}
+
+.btn-submit {
+  background-color: var(--color-text);
+  border: 0px solid white;
```
Comment (me-jterhorst) [inline src/pages/AddNewCake.css:12]: if it's a 0px border and you don't plan to transition it in some way, you could just write border: none;

File: src/pages/AddNewCake.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  padding: 0.5em 1.5em;
+  border-radius: 0.5em;
+  cursor: pointer;
+  font-size: 1.6em;
```
Comment (me-jterhorst) [inline src/pages/AddNewCake.css:16]: You might want to use rem instead of em

File: src/pages/AddNewCake.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import "./AddNewCake.css";
+
+export default function AddNewCake() {
+  return (
+    <form className="form">
```
Comment (me-jterhorst) [inline src/pages/AddNewCake.js:5]: This class Name seems redundant

File: src/components/Main.js (hunk 1 above)
Comment (borislavkb) [reply src/components/Main.js:5]: fair point ! changing these!

File: src/pages/AddNewCake.css (hunk 3 above)
Comment (borislavkb) [reply src/pages/AddNewCake.css]: Thanks ! Edited those units!

File: src/pages/AddNewCake.css (hunk 2 above)
Comment (borislavkb) [reply src/pages/AddNewCake.css]: implemented ! thx!

File: src/pages/AddNewCake.js (hunk 4 above)
Comment (borislavkb) [reply src/pages/AddNewCake.js]: It seems I was in a hurry there. The naming is corrected. :)

**Pull request merged**